### DISCUSSION TITLE
Browsing all radio stations from Radio Browser in HASS

### DISCUSSION THREAD

**Discussion by altserg** [music-assistant/support#3180] [Feature requests and ideas]
Hi all,

Is it possible to browse all radio stations provided by MASS Radio Browser in HASS ?
So far I only see the favorites in HASS, in order to find a radio I need to use 2 separate web sites - first MASS, and then HASS.
It's quite inconvenient

**Comment by OzGav** [comment]
No that is not currently possible.

**Reply by altserg** [reply]
Are there any plans to do so ?

**Reply by OzGav** [reply]
Not at the moment. I can change this to a feature request if you like and you can see how many votes you get.

**Reply by altserg** [reply]
It would be great. Currently I find it quite inconvenient to browse/search in 2 web sites for radio. Also it has security impact because MASS web site has to be reachable at least in local network. My idea was that I run it all locally on localhost/127.0.0.1, so only local connection between MASS and HASS is possible

**Reply by OzGav** [reply]
The intent is that you use the MA UI primarily. I will change this to a feature request.
